portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,16 @@ async function deleteComments() {
   const messagesElement = document.getElementById('comments-container');
   messagesElement.innerHTML = '';
 }
+
+function handleSubmit(e) {
+  e.preventDefault();
+  $.ajax({
+    type: 'POST',
+    url: '/data',
+    data: $(this).serialize(),
+    success: fetchContent(),
```

Comment:
are you sure this works? it should be `success: fetchContent,`

Reply:
Ah you're right, thank you!